python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -586,11 +586,13 @@ def run(self, exposure=None, selectSources=None, templateExposure=None, template
                 self.metadata.add("scaleTemplateVarianceFactor", templateVarFactor)
 
             if self.config.subtract.name == 'zogy':
-                subtractRes = self.subtract.subtractExposures(templateExposure, exposure,
-                                                              doWarping=True,
-                                                              spatiallyVarying=self.config.doSpatiallyVarying,
-                                                              doPreConvolve=self.config.doPreConvolve)
-                subtractedExposure = subtractRes.subtractedExposure
+                subtractRes = self.subtract.run(exposure, templateExposure, doWarping=True)
```

Comment:
I would also supply the keyword `calculateS=True` in the call to run, though see my comment in `ip_diffim` about renaming that keyword.